modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -239,10 +243,10 @@ Type::
 
 |===
 === .spec.agent.ipfix
-Description::
+Description::``
```

Comment:
these double back-ticks seem to have been added unintentionally, I don't see them in the source document - they cause a glitch in the spec.agent.ipfix rendering

Reply:
@jotak I noticed those yesterday and didn't push the change after I removed them. Pushed the removal of accidental backticks, so this is now addressed.